catalog/core/catalog-core-api/src/main/java/ddf/catalog/util/Describable.java

```diff
@@ -19,7 +19,8 @@
  *
  * @author ddf.isgs@lmco.com
  */
-public interface Describable {
+@Deprecated
+public interface Describable extends org.codice.ddf.platform.api.Describable {
```

Comment:
If we are deprecating this we should update all of it's implementors to use the new interface.